components/inputs/input-date.js

```diff
@@ -158,17 +161,24 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 		this.addEventListener('d2l-localize-behavior-language-changed', () => {
 			this._dateTimeDescriptor = getDateTimeDescriptorShared(true);
 			this.requestUpdate();
+			this._getContentWidth();
 		});
+
+		this._formattedValue = this.emptyStateText ? this.emptyStateText : '';
```

Comment:
How does this work for screen readers? Do the focus & blur events actually fire when the virtual cursor is being used such that it reads the placeholder instead of the empty text?

Reply:
So it mostly doesn't get read (voiceover seems to sometimes but I don't think NVDA ever did). Talking more to Jeff about it, he'd like it included using `aria-describedby`(when `empty-text` and no value selected). This seems like it would involve adding something like `description` to `input-text` in order to add an offscreen span with the text and `aria-describedby` to the `input` in `input-text`. Thoughts or concerns or alternatives to this?